docs/cpp/config-clang-mac.md

```diff
@@ -3,124 +3,143 @@ Order: 4
+You can also enable [Auto Save](/docs/editor/codebasics.md#saveauto-save) to automatically save your file changes, by checking **Auto Save** in the main **File** menu.
 
-Visual Studio code places these settings in `.vscode/c_cpp_properties.json`. If you open that file directly, it should look like this (depending on your specific path):
+The Activity Bar on the edge of Visual Studio Code lets you open different views such as **Search**, **Source Control**, and **Run**. You'll look at the **Run** view later in this tutorial. You can find out more about the other views in the VS Code [User Interface documentation](/docs/getstarted/userinterface.md).
 
-```json
-{
-    "configurations": [
-        {
-            "name": "macOS",
-            "includePath": [
-                "${workspaceFolder}/**"
-            ],
-            "defines": [],
-            "macFrameworkPath": [
-                "/System/Library/Frameworks",
-                "/Library/Frameworks"
-            ],
-            "compilerPath": "/usr/bin/clang",
-            "cStandard": "c11",
-            "cppStandard": "c++17",
-            "intelliSenseMode": "${default}"
-        }
-    ],
-    "version": 4
-}
-```
+>**Note**: When you save or open a C++ file, you may see a notification from the C/C++ extension about the availability of an Insiders version, which lets you test new features and fixes. You can ignore this notification by selecting the `X` (**Clear Notification**).
+
+## Explore IntelliSense
+
+In the `helloworld.cpp` file, hover over `vector` or `string` to see type information. After the declaration of the `msg` variable, start typing `msg.` You should immediately see a completion list that shows all the member functions, and a window that shows the type information for the `msg` object:
+
+![Statement completion IntelliSense](images/wsl/msg-intellisense.png)
+
+You can press the `kbstyle(Tab)` key to insert the selected member. Then, when you add the opening parenthesis, you'll see information about arguments that the function requires.
+
+## Build helloworld.cpp
+
+Next, you'll create a `tasks.json` file to tell VS Code how to build (compile) the program. This task will invoke the Clang C++ compiler to create an executable file from the source code.
 
-## Create a build task
+It's important to have `helloworld.cpp` open in the editor because the next step uses the active file in the editor as context to create the build task in the next step.
 
-Next, we need to create a `tasks.json` file to tell VS Code how to build (compile) the program. This task will invoke the Clang compiler to create an executable file based on the source code.
+From the main menu, choose **Terminal** > **Configure Default Build Task**.
+A dropdown will appear listing various predefined build tasks for the compilers that VS Code found on your machine. Choose **C/C++ clang++ build active file** to build the file that is currently displayed (active) in the editor.
```

Comment:
I think the drop down shows "C/C++: clang build active file".
At least this is what your screenshot shows.

Reply:
I got the wrong one. Fixed to show clang++